gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsTimeStatistic.java

```diff
@@ -0,0 +1,104 @@
+package com.google.cloud.hadoop.fs.gcs;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Iterators;
+import com.google.common.collect.Maps;
+import java.util.EnumSet;
+import org.apache.hadoop.classification.InterfaceStability;
+
+/** Time Statistics which are collected in GCS. */
+@InterfaceStability.Unstable
+public enum GhfsTimeStatistic {
+  /** Total Time for open(). */
+  OPEN("timeElapsedOpenMusec", "total time required for open in µ seconds", 0),
```

Comment:
consider renaming this to micro (also at other places

Reply:
Do you mean 'timeElapsedOpenMicrosec" ?